### PR TITLE
Bug 1830768 - Add nimbus flag for toolbar position

### PR DESCRIPTION
**Pull request by sarah541**
### Pull Request checklist

- [ ] **Quality**: This PR builds and passes detekt/ktlint checks (A pre-push hook is recommended)
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Changelog**: This PR includes [a changelog entry](https://github.com/mozilla-mobile/firefox-android/blob/main/docs/changelog.md) or does not need one
- [ ] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/firefox-android/blob/main/docs/shared/android/accessibility_guide.md) or does not include any user facing features

### After merge
- **Breaking Changes**: If this is a breaking Android Components change, please push a draft PR on [Reference Browser](https://github.com/mozilla-mobile/reference-browser) to address the breaking issues.

### To download an APK when reviewing a PR (after all CI tasks finished running):
1. Click on `Checks` at the top of the PR page.
2. Click on the `firefoxci-taskcluster` group on the left to expand all tasks.
3. Click on the `build-apk-{fenix,focus,klar}-debug` task you're interested in.
4. Click on `View task in Taskcluster` in the new `DETAILS` section.
5. The APK links should be on the right side of the screen, named for each CPU architecture.

### GitHub Automation
https://bugzilla.mozilla.org/show_bug.cgi?id=1830768

### CODE REVIEW

File: fenix/app/nimbus.fml.yaml [hunk 1]
```diff
@@ -142,6 +142,13 @@ import:
         }
 
 features:
+  toolbar:
+    description: The searchbar/awesomebar that user uses to search.
+    variables:
+      toolbar-position-top:
+        description: If true, toolbar appears at top of the screen.
+        type: Boolean
+        default: false
```
Comment (jhugman) [inline fenix/app/nimbus.fml.yaml:151]: If I remember correctly, you wanted to enable it for `nighty` and `developer` channels.

You can do that with:

```suggestion
        default: false
    defaults:
      - channel: nightly, developer
        value:
          toolbar-position-top: true
```

Comment (sarah541) [reply fenix/app/nimbus.fml.yaml:151]: That's right! I missed that. Thank you

**Review by rocketsroger — APPROVED** [review]

**Comment by sarah541** [comment]
@Mergifyio backport releases_v115

**Comment by DonalMe** [comment]
@mergify backport releases_v115

**Pull request merged**